packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.scss

```diff
@@ -138,4 +138,15 @@
     background: $ms-color-neutralTertiaryAlt;
     color: $ms-color-neutralDark;
   }
+}
+
+.suggestionsAvailable {
```

Comment:
@cschlechty is there a preferred method for hiding text used for the sole purpose of alerts? I thought we'd done that in some other components.